src/App.tsx

```diff
@@ -1,63 +1,109 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+import { GoodList } from './Components/GoodList/GoodList';
+import { goodsFromServer } from './Components/goodListData';
+
+enum SortField {
+  Alphabet = 'abc',
+  Id = 'id',
+  Default = '',
+}
+
+interface FilterSort {
+  sortField: SortField;
+  isReversed: boolean;
+}
+
+function getPreparedGoods(
+  goods: string[],
+  {
+    sortField,
+    isReversed,
+  }: FilterSort,
+) {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods = preparedGoods.sort((good1, good2) => {
```

Comment:
```suggestion
    preparedGoods.sort((good1, good2) => {
```

sort mutate or not mutate array?